UI/Pages/Game.razor

```diff
@@ -34,6 +34,18 @@
           <LoserScreen Loser="@_host"/>
         }
       }
+      else if (_host != null && !_host.HasTurn && _guest != null && !_guest.HasTurn)
```

Comment:
Instead of checking their turns, it makes more sense to me to check the `HasWon` property. If `HasWon` is `false` on both, the host and the guest, then its a draw.

Reply:
good point, that was my initial approach but then I found out that HasWon is false on both players by default, so the game went straight to the draw screen. instead, I used this approach based on the conditional logic we currently have.